api/src/socketio.ts

```diff
@@ -86,26 +110,45 @@ export class Socketio {
         const changeReq = plainToInstance(ChangeReqDto, data);
```

Comment:
As a general not this file is growing quite a bit now so maybe in a follow-up PR some parts could be extracted, like the validation, which might make testing those specific parts easier as well as they don't rely on socket.io.